content/operate/kubernetes/reference/redis_enterprise_remote_cluster_api.md

```diff
@@ -0,0 +1,167 @@
+---
+title: RedisEnterpriseRemoteCluster API Reference
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+linkTitle: RERC API
+weight: 30
+---
+
+API groups:
+
+
+- [app.redislabs.com/v1alpha1](#appredislabscomv1alpha1)
+
+
+
+
+# app.redislabs.com/v1alpha1
+
+
+
+
+RedisEntepriseRemoteCluster represents a remote participating cluster.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+      <td><b>apiVersion</b></td>
+      <td>string</td>
+      <td>app.redislabs.com/v1alpha1</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b>kind</b></td>
+      <td>string</td>
+      <td>RedisEnterpriseRemoteCluster</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b><a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#objectmeta-v1-meta">metadata</a></b></td>
+      <td>object</td>
+      <td>Refer to the Kubernetes API documentation for the fields of the `metadata` field.</td>
+      <td>true</td>
+      </tr><tr>
+        <td><b><a href="#spec">spec</a></b></td>
+        <td>object</td>
+        <td>
+          <br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#status">status</a></b></td>
+        <td>object</td>
+        <td>
+          <br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+### spec
+<sup><sup>[↩ Parent](#)</sup></sup>
+
+
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>apiFqdnUrl</b></td>
+        <td>string</td>
+        <td>
+          The URL of the cluster, will be used for the active-active database URL.<br/>
```

Comment:
```suggestion
          The URL of the cluster that will be used for the Active-Active database URL.<br/>
```

Reply:
Unfortunately, on the API docs, the copy editing needs to be done in the source code because these are auto-generated. But I do have copy editing these as my next task.